src/patternfly/components/DataToolbar/data-toolbar-item-overflow-menu.hbs

```diff
@@ -0,0 +1,6 @@
+<div class="pf-c-data-toolbar__item pf-m-overflow-menu{{#if data-toolbar-overflow-menu--modifier}} {{data-toolbar-overflow-menu--modifier}}{{/if}}"
```

Comment:
Can you use {{#> data-toolbar-item}} here and `concat()` `pf-m-overflow-menu` and `data-toolbar-overflow-menu--modifier` in `data-toolbar-item--modifier`?